### DISCUSSION TITLE
Unable to open and build the WinUI 3 source code

### DISCUSSION THREAD

**Discussion by yvan-noume** [microsoft/microsoft-ui-xaml#9738] [Q&A] [answered]
I've cloned the WinUI 3 repository locally but I'm unabled to open the source code (with Visual studio) and build it. Here is the link for that repo: https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.5-stable.
The solution that I tried to open is: "...\microsoft-ui-xaml\dxaml\Microsoft.UI.Xaml.sln"

**How to reproduce the problem:** Clone the WinUI 3 repository (https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.5-stable) and open the solution mentioned above ("...\microsoft-ui-xaml\dxaml\Microsoft.UI.Xaml.sln") with visual studio.

**Expected behaviour:** All the projects of the solution are loaded.

**Actual behaviour:** No project is loaded. I got asked about credentials. And when I get past the credentials step, still no project is loaded.

**Comment by codendone** [comment, marked as answer]
From https://github.com/microsoft/microsoft-ui-xaml/issues/9682#issuecomment-2174533624:
> ### Q: ​Will we be able to build from the source or only view due to private dependencies?
> #### A: To be clear, we are doing shared source, not open source, as there are underlying interwoven components linking WinUI 3 to the system that we can't share. We will not support building, deploying, or running the source code and will not accept PRs for it. We will support viewing, searching, and debugging of the source code when it is available.

That info is also in [README.md in the winui3 branch](https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/main/README.md#contributing-to-winui):
> **Please note that although the source code for WinUI 3 is available in this repository it cannot be compiled by non-Microsoft developers at this time.**

**Reply by yvan-noume** [reply]
Thanks  for the clarification.
